packages/hoprd/hopr-admin/pages/index.js

```diff
@@ -41,12 +82,14 @@ export default function Home() {
         <input
           id="command"
           type="text"
-          disabled={!isNodeReady}
+          // disabled={!isNodeReady}
```

Comment:
Probably an error?

Reply:
No, it's not.  I should have removed this line